pybaseball/pitching_leaders.py

```diff
@@ -1,50 +1,34 @@
-_FG_PITCHING_LEADERS_URL = "/leaders.aspx?pos=all&stats=pit&lg={league}&qual={qual}&type={types}&season={end_season}&month=0&season1={start_season}&ind={ind}&team=&rost=&age=&filter=&players=&page=1_100000"
+from pybaseball.datasources.fangraphs import FanGraphsLeague, fg_pitching_data
 
 
-def pitching_stats(start_season: int, end_season: int = None, league: str = 'all', qual: Union[int, str] = 1,
-                   ind: int = 1):
+def pitching_stats(start_season: int, end_season: int = None, league: str = 'all', qual: Optional[int] = None,
+                   ind: int = 1) -> pd.DataFrame:
     """
     Get season-level pitching data from FanGraphs.
 
     ARGUMENTS:
-    start_season : int      : first season you want data for (or the only season if you do not specify an end_season)
-    end_season   : int      : final season you want data for
-    league       : str      : "all", "nl", or "al"
-    qual         : int, str : minimum number of pitches thrown to be included in the data (integer).
-                              Use the string 'y' for fangraphs default.
-    ind          : int      : 1 if you want individual season-level data
-                              0 if you want a player's aggreagate data over all seasons in the query
+    start_season : int           : first season you want data for
+                                   (or the only season if you do not specify an end_season)
+    end_season   : int           : final season you want data for
+    league       : str           : "all", "nl", or "al"
+    qual         : Optional[int] : minimum number of plate appearances to be included in the data.
+                                   If None is specified, the FanGraphs default ('y') is used.
+                                   Default = None
+    ind          : int           : 1 if you want individual season-level data
+                                   0 if you want a player's aggreagate data over all seasons in the query
```

Comment:
this could also be bool, if we're going to change the API around here (and that wouldn't break code that used 0/1)

Reply:
It's true python would just use the truthiness of the 1 or 0, so the effect SHOULD be the same, for anyone who uses MyPy against their code to ensure data type consistency, that would raise an error.

I was erring on the side of not breaking existing function declarations for end user consistency.